src/TextArea/TextArea.less

```diff
@@ -0,0 +1,98 @@
+@import (reference) "../less/colors";
+@import (reference) "../less/border";
+@import (reference) "../less/sizing";
+@import (reference) "../less/type-size";
+@import (reference) "../less/type-utilities";
+@import (reference) "../less/spacing";
+
+.TextArea {
+  .padding--left--s;
+  .padding--top--3xs;
+  .text--regular;
+  .border--s(@neutral_silver);
+  position: relative;
+  color: @neutral_black;
+  background: @neutral_white;
+  box-sizing: border-box;
+  width: 100%;
+
+  .TextArea--infoRow {
+    .text--tiny;
+    .text--caps;
+    position: absolute;
+    top: @size_xs;
+    width: 100%;
+    color: @neutral_dark_gray;
+
+    .TextArea--label {
+      left: @size_s;
+      opacity: 1;
+    }
+
+    .TextArea--required {
+      position: absolute;
+      right: @size_l;
+    }
+
+    .TextArea--error {
+      color: @alert_red;
+      position: absolute;
+      right: @size_l;
+    }
+  }
+
+  .TextArea--input {
+    .text--medium;
+    display: block;
+    width: 100%;
+    border: 1px solid transparent;
+    resize: vertical;
+    line-height: @size_2xl;
+    min-height: @size_2xl;
+    padding-top: @size_s;
+    background-color: transparent;
+    outline: none;
+  }
+
+  &.TextArea--placeholder-shown {
+    label { opacity: 0 !important; }
```

Comment:
Oh, do you mean new line for opacity? I copied this from TextInput.less but it looks like Select.less put it on its own line.